draft-ietf-scitt-architecture.md

```diff
@@ -980,52 +970,18 @@ So long as actors maintain proper control of their signing keys and identity inf
-This ensures any Relying Party that trusts at least one such Auditor that (2, 3) will be blamed to the Transparency Service.
-
-Due to the operational challenge of maintaining a globally consistent Verifiable Data Structure, some Transparency Services may provide limited support for historical queries on the Signed Statements they have registered and accept the risk of being blamed for inconsistent Registration or Issuer Equivocation.
-
-Relying Parties and Auditors may also witness (1, 4) but may not be able to collect verifiable evidence for it.
-
-### Availability of Receipts
-
-Networking and Storage are trusted only for availability.
-
-Auditing may involve access to data beyond what is persisted in the Transparency Services.
-For example, the registered Transparency Service may include only the hash of a detailed SBOM, which may limit the scope of auditing.
-
-Resistance to denial-of-service is implementation specific.
-
-Actors may want to independently keep their own record of the Signed Statements they issue, endorse, verify, or audit.
+The security considerations for specific verifiable data structures are out of scope for this document.
+See {{-RECEIPTS}} for the generic security considerations that apply to Verifiable Data Structure and Receipts.
 
 ### Cryptographic Agility
 
 The SCITT Architecture supports cryptographic agility.
-The actors depend only on the subset of signing and Receipt schemes they trust.
-This enables the gradual transition to stronger algorithms, including e.g. post-quantum signature algorithms.
-
-### Transparency Service Client Applications
-
-Authentication of Client applications is out of scope for this document.
-Transparency Services MAY authenticate Client applications.
-The specification of authentication and authorization policies is out of scope for this document.
-
-### Impersonation
+There are no mandatory to implement signing algorithms for Signed Statements or Receipts.
 
-The identity resolution mechanism is trusted to associate long-term identifiers with their public signature-verification keys.
-Transparency Services and other parties may record identity-resolution evidence to facilitate its auditing.
+### Key Compromise
 
-If one of the credentials of an Issuer gets compromised, the SCITT Architecture still guarantees the authenticity of all Signed Statements signed with this credential that have been registered on a Transparency Service before the compromise.
-It is up to the Issuer to notify Transparency Services of credential revocation to stop Relying Parties from accepting Signed Statements signed with compromised credentials.
+Revocation strategies for compromised keys are out of scope for this document.
+It is important for Issuers and Transparency Services to clearly communicate when keys are compromised, so that Signed Statements can be rejected by Transparency Services, or Receipts can be ignored by Relying Parties.
```

Comment:
```suggestion
It is important for Issuers and Transparency Services to clearly communicate when keys are compromised, so that Signed Statements can be rejected by Transparency Services or Receipts can be ignored by Relying Parties.
```